src/data/ulift.lean

```diff
@@ -24,11 +25,20 @@ instance [unique α] : unique (plift α) := equiv.plift.unique
 instance [decidable_eq α] : decidable_eq (plift α) := equiv.plift.decidable_eq
 instance [is_empty α] : is_empty (plift α) := equiv.plift.is_empty
 
+lemma up_injective : injective (@up α) := equiv.plift.symm.injective
+lemma up_surjective : surjective (@up α) := equiv.plift.symm.surjective
+lemma up_bijective : bijective (@up α) := equiv.plift.symm.bijective
+
+@[simp] lemma up_inj {x y : α} : up x = up y ↔ x = y := up_injective.eq_iff
+
+lemma down_surjective : surjective (@down α) := equiv.plift.surjective
+lemma down_bijective : bijective (@down α) := equiv.plift.bijective
```

Comment:
Any reason for not adding the injective version?

Reply:
It already exists elsewhere.